notes/relaease.md

```diff
@@ -0,0 +1,51 @@
+# Release (WIP)
```

Comment:
This note is a still WIP, but I wrote up all things which I know about the release.

Reply:
This is great! Thank you so much for putting this together :)